dist/index.js

```diff
@@ -49860,7 +49860,7 @@ const createGitHubRepository = fp_ts_TaskEither__WEBPACK_IMPORTED_MODULE_2__.try
                 ...defaults,
                 ref: `heads/${branch}`,
                 sha: commit.sha,
-                force: true,
+                force: false,
```

Comment:
nits. Backward compatibility will be lost, and we would like to achieve full file synchronization in this action. For this reason, we would like to see a `force` option added :+1: